components/org.wso2.carbon.identity.recovery.endpoint/src/main/java/org/wso2/carbon/identity/recovery/endpoint/impl/RecoverPasswordApiServiceImpl.java

```diff
@@ -81,6 +81,11 @@ public Response recoverPasswordPost(RecoveryInitiatingRequestDTO recoveryInitiat
         } catch (IdentityRecoveryException e) {
             RecoveryUtil.handleInternalServerError(Constants.SERVER_ERROR, e.getErrorCode(), LOG, e);
         } catch (Throwable throwable) {
+            if (throwable != null && StringUtils.equals(Constants.ERROR_MESSAGE_EMAIL_NOT_FOUND,
+                    throwable.getMessage())) {
+                LOG.error(throwable);
```

Comment:
```suggestion
                LOG.error(throwable.getMessage(), throwable);
```

Reply:
Fixed in https://github.com/Geesa-Vihara/identity-governance/commit/88b15bf0fe93caf31f17c51edc8686a9941539cf